Caused by: java.lang.NoClassDefFoundError: org/gradle/tooling/BuildCancelledException

```
Caused by: java.lang.NoClassDefFoundError: org/gradle/tooling/BuildCancelledException
	at java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
	at java.lang.Class.getDeclaredConstructors(Class.java:2020)
	at org.codehaus.groovy.reflection.CachedClass$2.lambda$initValue$4(CachedClass.java:71)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.codehaus.groovy.reflection.CachedClass$2.initValue(CachedClass.java:76)
	at org.codehaus.groovy.reflection.CachedClass$2.initValue(CachedClass.java:66)
	at org.codehaus.groovy.util.LazyReference.getLocked(LazyReference.java:50)
	at org.codehaus.groovy.util.LazyReference.get(LazyReference.java:37)
	at org.codehaus.groovy.reflection.CachedClass.getConstructors(CachedClass.java:259)
	at groovy.lang.MetaClassImpl.<init>(MetaClassImpl.java:223)
	at groovy.lang.MetaClassImpl.<init>(MetaClassImpl.java:233)
	at groovy.lang.MetaClassRegistry$MetaClassCreationHandle.createNormalMetaClass(MetaClassRegistry.java:166)
	at groovy.lang.MetaClassRegistry$MetaClassCreationHandle.createWithCustomLookup(MetaClassRegistry.java:156)
	at groovy.lang.MetaClassRegistry$MetaClassCreationHandle.create(MetaClassRegistry.java:139)
	at org.codehaus.groovy.reflection.ClassInfo.getMetaClassUnderLock(ClassInfo.java:272)
	at org.codehaus.groovy.reflection.ClassInfo.getMetaClass(ClassInfo.java:315)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.getMetaClass(MetaClassRegistryImpl.java:258)
	at org.codehaus.groovy.runtime.InvokerHelper.invokeConstructorOf(InvokerHelper.java:1059)
	at org.codehaus.groovy.runtime.DefaultGroovyMethods.newInstance(DefaultGroovyMethods.java:17679)
	at org.grails.cli.profile.AbstractProfile.initialize(AbstractProfile.groovy:135)
	at org.grails.cli.profile.repository.AbstractJarProfileRepository$JarProfile.<init>(AbstractJarProfileRepository.groovy:120)
	at org.grails.cli.profile.repository.AbstractJarProfileRepository.registerProfile(AbstractJarProfileRepository.groovy:99)
	at org.grails.cli.profile.repository.MavenProfileRepository.processUrls(MavenProfileRepository.groovy:118)
	at org.grails.cli.profile.repository.MavenProfileRepository.getAllProfiles(MavenProfileRepository.groovy:153)
	at sun.reflect.GeneratedMethodAccessor228.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.MethodMetaProperty$GetBeanMethodMetaProperty.getProperty(MethodMetaProperty.java:76)
	at org.codehaus.groovy.runtime.callsite.GetEffectivePogoPropertySite.callGetProperty(GetEffectivePogoPropertySite.java:48)
	at profiles.run(profiles.groovy:9)
	at profiles$run.call(Unknown Source)
	at appgenerator.ScriptExecutor.executeScript(ScriptExecutor.groovy:47)
	at appgenerator.ScriptExecutor.executeScript(ScriptExecutor.groovy)
	at appgenerator.ProfileService.getProfiles(ProfileService.groovy:98)
	at appgenerator.$ProfileService$Definition$Intercepted.$$access$$getProfiles(Unknown Source)
	at appgenerator.$ProfileService$Definition$Exec.dispatch(Unknown Source)
	at io.micronaut.context.AbstractExecutableMethodsDefinition$DispatchedExecutableMethod.invoke(AbstractExecutableMethodsDefinition.java:338)
	at io.micronaut.aop.chain.MethodInterceptorChain.proceed(MethodInterceptorChain.java:128)
	at io.micronaut.cache.interceptor.CacheInterceptor.doContextProceed(CacheInterceptor.java:722)
	at io.micronaut.cache.interceptor.CacheInterceptor.doProceed(CacheInterceptor.java:726)
	at io.micronaut.cache.interceptor.CacheInterceptor.interceptSync(CacheInterceptor.java:416)
	at io.micronaut.cache.interceptor.CacheInterceptor.intercept(CacheInterceptor.java:156)
	at io.micronaut.aop.chain.MethodInterceptorChain.proceed(MethodInterceptorChain.java:137)
	at appgenerator.$ProfileService$Definition$Intercepted.getProfiles(Unknown Source)
	at appgenerator.ProjectOptionsService.profileListByProjectType(ProjectOptionsService.groovy:40)
	at appgenerator.ProjectOptionsService.defaultProjectOptions(ProjectOptionsService.groovy:53)
	at appgenerator.IndexController.index(IndexController.groovy:20)
	at org.grails.core.DefaultGrailsControllerClass$MethodHandleInvoker.invoke(DefaultGrailsControllerClass.java:223)
	at org.grails.core.DefaultGrailsControllerClass.invoke(DefaultGrailsControllerClass.java:188)
	at org.grails.web.mapping.mvc.UrlMappingsInfoHandlerAdapter.handle(UrlMappingsInfoHandlerAdapter.groovy:90)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1067)
	... 47 common frames omitted
```

--- a/build.gradle
+++ b/build.gradle
@@ -85,6 +85,7 @@ dependencies {
     compileOnly "io.micronaut:micronaut-inject-groovy"
 
     profile "org.grails.profiles:web"
+    runtimeOnly "org.gradle:gradle-tooling-api:7.0"
     runtimeOnly "org.glassfish.web:el-impl:2.1.2-b03"
     runtimeOnly "javax.xml.bind:jaxb-api:2.3.1"
     runtimeOnly "com.bertramlabs.plugins:asset-pipeline-grails:3.3.4"
@@ -99,7 +100,6 @@ dependencies {
     testImplementation "org.seleniumhq.selenium:selenium-remote-driver:$seleniumVersion"
     testImplementation "org.seleniumhq.selenium:selenium-api:$seleniumVersion"
     testImplementation "org.seleniumhq.selenium:selenium-support:$seleniumVersion"
-    testRuntimeOnly "org.gradle:gradle-tooling-api:7.0"
     testRuntimeOnly "org.seleniumhq.selenium:selenium-chrome-driver:$seleniumVersion"
     testRuntimeOnly "org.seleniumhq.selenium:selenium-firefox-driver:$seleniumVersion"
 

